ui.tests/test-module/libs/commons/guideSelectors.js

```diff
@@ -183,6 +186,9 @@ var selectors = {
             HIDE : "coral-selectlist [value='HIDE_STATEMENT']",
             IS_CLICKED : "coral-selectlist [value='is clicked']",
             NAVIGATE_TO : "coral-selectlist [value='NAVIGATE_TO']"
+        },
+        ruleSummary : {
+            CREATED_RULE: "#rule-summary table[handle='table'] tr[title='Button - Click']"
```

Comment:
Can we use specific CSS selectors and not rely on coral, this might break on 650